### ISSUE TITLE
[BUG] Brackets inside img src are automatically encoded when rendering

### ISSUE THREAD

**Issue by psociety** [bootstrap-email/bootstrap-email#160]
Again, TY FOR CREATING THIS TOOL!!!

#### What version of Bootstrap Email are you using?
v1.1.2 -> https://app.bootstrapemail.com/

#### How are you compiling?
Online editor

#### Describe your issue

Having the following code:
```html
<img src="{{ full_url_for('assets') }}/img/mail/test.png">
<a href="{{ mainLink }}">Say hello!</a>
```

It will result in:
```html
<img src="%7B%7B%20full_url_for('assets')%20%7D%7D/img/mail/test.png">
<a href="{{ mainLink }}">Say hello!</a>
```

**Comment by stuyam** [comment]
Ahh I see what is happening. The functionality works the same inside `href` and `src` attributes however the difference here is one is the `src` example has `/img/mail/test.png` trailing the `}}` before the closing quotation. Totally valid use case the library just doesn't account for that so need work on allowing leading and trailing text surrounding the brackets. Should be a fairly straightforward change to the regex here: https://github.com/bootstrap-email/bootstrap-email/blob/master/lib/bootstrap-email/converters/support_url_tokens.rb will take a look at some point.

**Comment by stuyam** [comment]
Fixed in latest version [1.1.3](https://github.com/bootstrap-email/bootstrap-email/releases/tag/1.1.3)
